src/Sentry/Extensibility/DisabledHub.cs

```diff
@@ -87,11 +87,15 @@ public void BindException(Exception exception, ISpan span)
     /// <summary>
     /// Returns null.
     /// </summary>
-    public TransactionContext? ContinueTrace(
+    public TransactionContext ContinueTrace(
         SentryTraceHeader? traceHeader,
         BaggageHeader? baggageHeader,
         string? name = null,
-        string? operation = null) => null;
+        string? operation = null)
+    {
+        // Transactions from DisabledHub are always sampled out
+        return new TransactionContext(string.Empty, string.Empty, false);
```

Comment:
I think we could do the same [thing we've done for the `DisabledHub.Instance`](https://github.com/getsentry/sentry-dotnet/blob/dfc4274fc79cdd1472649387bee6feee713343cf/src/Sentry/Extensibility/DisabledHub.cs#L11) here, to avoid creating a new object every time this method is called.

Not sure where to put that... maybe `TransactionContext.Disabled` or `TransactionContext.DisabledInstance`?